HabitTracker.Wolffles/Habit.cs

```diff
@@ -3,10 +3,10 @@
 namespace habit_tracker;
 class Habit : IHabit
 {
-	private string m_connectionString = "";
-	private string m_habitName = "";
-	private string m_habitTable = "";
-	private string m_habitUnit = "";
+	public string m_connectionString { get; set; }
```

Comment:
If you check that link I gave you for properties you will notice that the first letter is capitalized. It's kind of a naming convention e.g. `public string FirstName { get; set;}`

I also want to point out that since you don't re-assign any of these properties outside the Habit class you can just have a get and remove the set e.g. `public string FirstName { get; }`

Reply:
Interesting. Is it a C# specific convention? Often times in C++ I was encouraged to start all my members in a class with m_ lower case.

Reply:
I'm also going to look into SQL Injection but most importantly wanted to say thank you for the reviews! They've been very helpful and I appreciate the work you guys do!

Reply:
> Interesting. Is it a C# specific convention? Often times in C++ I was encouraged to start all my members in a class with m_ lower case. 

Yep it is. When you can you should check [Common C# Coding Conventions](https://learn.microsoft.com/en-us/dotnet/csharp/fundamentals/coding-style/coding-conventions)